src/BannerViewManager.js

```diff
@@ -32,6 +32,7 @@ const getSizeForType = (type: AdType) => sizeForType[type] || sizeForType.standa
 type BannerViewProps = {
   type: AdType,
   placementId: string,
+  style: { [key: string]: any },
```

Comment:
It should be `number|string|{[key: string]: number|string}`.
@satya164 Am I right?